src/Analysim.Web/Controllers/AccountController.cs

```diff
@@ -228,16 +231,39 @@ public async Task<IActionResult> Register([FromForm] AccountRegisterVM formdata)
             // Hold Error List
             List<string> errorList = new List<string>();
 
+            string registrationSurvey = formdata.RegistrationSurvey;
```

Comment:
You can convert this from/to JSON.

Reply:
Hi @cengique.

I could not find any way to add json field in the migration file. I decided to leave registrationSurvey as a string field in the User class. I converted the string to json format like this:

let registrationSurvey = {"workPlace": userReg.workPlace,"positionTitle":userReg.positionTitle,"whereDidYouHearAboutAnalysim": userReg.whereDidYouHearAboutAnalysim,"wouldYouLikeToHearUpdatesFromUs": userReg.wouldYouLikeToHearUpdatesFromUs,"registrationCode":userReg.registrationCode};

I stringified it before sending to the backend.

 this.accountService.register(userReg.username, userReg.password, userReg.emailAddress,JSON.stringify(registrationSurvey))

Reply:
@KshitizSareen I think this works!